__tests__/Unit/Components/Tasks/ProgressUpdateCard/ProgressUpdateCardPresentation.test.tsx

```diff
@@ -123,7 +129,76 @@ describe('ProgressUpdateCardPresentation Component', () => {
             onCardClick: mockedOnCardClick,
         };
     });
+
+    it('should rotate the angle icon when expanded', () => {
+        (useRouter as jest.Mock).mockReturnValue({
+            query: { dev: 'false' },
+        });
```

Comment:
Why is this being mocked here?

Reply:
As these changes are currently behind a feature flag, we need to mock `useRouter` to set the dev query parameter to true for testing.